### DISCUSSION TITLE
IOS steps are not updating immediately . its taking 10 mins to update

### DISCUSSION THREAD

**Discussion by RonakBansal321** [doville/rn-fitness-tracker#132] [Q&A]
please suggest any help below is my code
verison using rn-fitness-tracker@1.5.4

 const authorizationStatusios = await FitnessTrackerAPI.setupTracking([
            HealthDataTypes.stepsCount,
          ]);
this.setState({ auth: authorizationStatusios });
          // // Get steps total today
          if (authorizationStatusios.authorized) {
            const steps = await FitnessTrackerAPI.getStepsToday();
            this.stepsCount = steps;
            console.log("1121212121212121212121212121", steps);
}

**Comment by alarm109** [comment]
You can check the Health app and see that data takes some time to update. And this package takes data from apple Health. 

If you need instant instant feedback, you can try to use PedometerApi, which is only supported in version 1.x. It should give instant feedback.
